jaxlib/mosaic/gpu/custom_call.cc

```diff
@@ -415,6 +417,40 @@ GetKernelCache() {
   return std::make_pair(&context_cache, &mutex);
 }
 
+absl::StatusOr<std::pair<std::string, std::string>> GetHostAndInitFuncNames(
+    mlir::ModuleOp module_op) {
+  // We look for two top level C-interface functions:
+  // - "host" function with symbol name "_mlir_ciface_<foo>"
+  // - "init" function with symbol name "_mlir_ciface_<foo>_init"
+  const std::string prefix = "_mlir_ciface_";
```

Comment:
Nit: can this be a `constexpr std::string_view`?

Reply:
Fixed.